JoelFaldin/ServerSort-MuniCrud-Backend
# Testing/directions service test

# Add tests for directions service

## Description 🗒️✏️

Added tests for directions.service. Now all services are correctly tested :D

## How to test it? 🎯⚗️🔬

- Install the dependencies using `pnpm i`.
- For prisma, run `pnpx prisma migrate dev --name init`, then run `pnpx prisma generate`. Aditionally, you can start a local page to inspect the tables using `pnpx prisma studio`, but is not necessary for the tests.
- Run `pnpm run test`, and you should see something like this:
![image](https://github.com/user-attachments/assets/97ca0d07-d578-4b18-b7fd-c528fd7715bf)
- Alternatively, you can run `pnpm run test -- src/directions/__tests__/` to run tests in the `directions` folder.
